tests/backup/backup_helper_test.go

```diff
@@ -0,0 +1,1205 @@
+func ShareBackupWithUsersAndAccessAssignment(backupNames []string, users []string, ctx context.Context) (map[userAccessContext]string, error) {
+	log.InfoD("Sharing backups with users with different access level")
+	accessUserBackupContext := make(map[userAccessContext]string)
+	var err error
+	var ctxNonAdmin context.Context
+	var access BackupAccess
+	for i, user := range users {
+		userIndex := i % 3
+		switch userIndex {
+		case 0:
+			access = ViewOnlyAccess
+			err = ShareBackup(backupNames[i], nil, []string{user}, access, ctx)
+		case 1:
+			access = RestoreAccess
+			err = ShareBackup(backupNames[i], nil, []string{user}, access, ctx)
+		case 2:
+			access = FullAccess
+			err = ShareBackup(backupNames[i], nil, []string{user}, access, ctx)
+		default:
+			access = ViewOnlyAccess
+			err = ShareBackup(backupNames[i], nil, []string{user}, access, ctx)
+		}
+		if err != nil {
+			return accessUserBackupContext, fmt.Errorf("unable to share backup %s with user %s Error: %v", backupNames[i], user, err)
+		}
+		ctxNonAdmin, err = backup.GetNonAdminCtx(users[i], "Password1")
+		if err != nil {
+			return accessUserBackupContext, fmt.Errorf("unable to get user context: %v", err)
+		}
+		accessContextUser := userAccessContext{users[i], access, ctxNonAdmin}
+		accessUserBackupContext[accessContextUser] = backupNames[i]
+	}
+	return accessUserBackupContext, nil
+}
+
+// GetAllBackupsAdmin returns all the backups that px-central-admin has access to
+func GetAllBackupsAdmin() ([]string, error) {
+	var bkp *api.BackupObject
+	backupNames := make([]string, 0)
+	backupDriver := Inst().Backup
+	ctx, err := backup.GetAdminCtxFromSecret()
+	if err != nil {
+		return nil, err
+	}
+	bkpEnumerateReq := &api.BackupEnumerateRequest{
+		OrgId: orgID}
+	curBackups, err := backupDriver.EnumerateBackup(ctx, bkpEnumerateReq)
+	if err != nil {
+		return nil, err
+	}
+	for _, bkp = range curBackups.GetBackups() {
+		backupNames = append(backupNames, bkp.GetName())
+	}
+	return backupNames, nil
+}
+
+// GetAllRestoresAdmin returns all the backups that px-central-admin has access to
+func GetAllRestoresAdmin() ([]string, error) {
+	restoreNames := make([]string, 0)
+	backupDriver := Inst().Backup
+	ctx, err := backup.GetAdminCtxFromSecret()
+	if err != nil {
+		return restoreNames, err
+	}
+
+	restoreEnumerateRequest := &api.RestoreEnumerateRequest{
+		OrgId: orgID,
+	}
+	restoreResponse, err := backupDriver.EnumerateRestore(ctx, restoreEnumerateRequest)
+	if err != nil {
+		return restoreNames, err
+	}
+	for _, restore := range restoreResponse.GetRestores() {
+		restoreNames = append(restoreNames, restore.Name)
+	}
+	return restoreNames, nil
+}
+
+// TODO: There is no delete org API
+/*func DeleteOrganization(orgID string) {
+	Step(fmt.Sprintf("Delete organization [%s]", orgID), func() {
+		backupDriver := Inst().Backup
+		req := &api.Delete{
+			CreateMetadata: &api.CreateMetadata{
+				Name: orgID,
```

Comment:
remove the commented part of not required

Reply:
done